api/clients/v2/payloadretrieval/validator_payload_retriever.go

```diff
@@ -127,7 +126,7 @@ func (pr *ValidatorPayloadRetriever) retrieveBlobWithTimeout(
 	blobKey corev2.BlobKey,
 	blobVersion corev2.BlobVersion,
 	blobCommitments encoding.BlobCommitments,
-	referenceBlockNumber uint32,
+	referenceBlockNumber uint64,
```

Comment:
Wait what? Why did this change to u64?? The contracts seem to always have expected a u32

Reply:
hmm but the retrieval clients assumed u64 which at the time appeared to avoid less downstream casting. Don't think there'd ever be a down casting bug though since no chain has an expected block # > 2^32